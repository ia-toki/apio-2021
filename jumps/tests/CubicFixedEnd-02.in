209636825ecd2c38299498cd26890ef71dd128c2
39 24
13 16 9 23 5 4 30 3 6 24 26 19 1 29 17 37 34 11 35 14 20 32 38 15 28 31 22 21 12 10 39 7 25 27 2 18 33 8 36
0 0 38 38
11 32 37 37
5 8 31 31
0 9 12 12
31 33 35 35
1 1 20 20
19 28 30 30
1 4 38 38
17 17 20 20
4 37 38 38
4 15 30 30
9 14 18 18
17 24 35 35
5 11 34 34
6 24 28 28
4 9 20 20
28 33 34 34
19 31 34 34
11 31 35 35
3 23 30 30
11 11 27 27
33 34 36 36
5 7 18 18
18 22 27 27
