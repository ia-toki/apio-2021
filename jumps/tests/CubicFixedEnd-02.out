fc5005cd102fffc504bc1615010ebf88823d65f3
OK
-1
-1
-1
-1
-1
-1
1
-1
-1
1
2
-1
-1
-1
-1
-1
-1
-1
-1
1
-1
1
-1
-1
